### PR TITLE
Fix missing imports and `cupy.show_config`

### PR DESCRIPTION
**Pull request by emcastillo**
After creating `cuda_backends` some imports where not changed and `cupy.show_config` wasn't able to detect cudnn, cutensor and other libraries

### CODE REVIEW

File: cupyx/runtime.py [hunk 1]
```diff
@@ -10,12 +10,12 @@
...
 
 try:
-    import cupy.cuda.cudnn as cudnn
+    import cupy_backends.cuda.libs.cudnn as cudnn
 except ImportError:
     cudnn = None
 
 try:
-    import cupy.cuda.nccl as nccl
+    import cupy_backends.cuda.libs.nccl as nccl
```
Comment (leofang) [inline cupyx/runtime.py:18]: I thought `nccl` is still in `cupy.cuda`?

**Comment by emcastillo** [comment]
Updated, PTAL

**Comment by leofang** [comment]
Jenkins, test this please

File: cupy/cuda/__init__.py [hunk 2]
```diff
@@ -51,6 +51,12 @@
 except ImportError:
     nccl_enabled = False
 
+try:
+    from cupy_backends.cuda.libs import cudnn  # NOQA
```
Comment (kmaehashi) [inline cupy/cuda/__init__.py:55]: This is not needed as cudnn is not exposed here.
https://github.com/cupy/cupy/blob/v8.0.0b4/cupy/cuda/__init__.py

Comment (emcastillo) [reply cupy/cuda/__init__.py]: Previous versions had `import cupy.cuda.cudnn as cudnn` as a valid path.
I think we should add a `cupy.cuda.cudnn` file that imports `cupy_backends.cuda.libs.cudnn` not to break compatibility?

**Comment by leofang** [comment]
Jenkins, test this please

**Pull request merged**